src/app/item-popup/ItemStats.tsx

```diff
@@ -66,30 +79,38 @@ function ItemStatRow({
 }) {
   const value = stat.value;
   const compareStatValue = compareStat ? compareStat.value : 0;
+
   const isMasterworkedStat =
     item.isDestiny2() && item.masterworkInfo && stat.statHash === item.masterworkInfo.statHash;
   const masterworkValue =
     (item.isDestiny2() && item.masterworkInfo && item.masterworkInfo.statValue) || 0;
-  const higherLowerClasses = {
+
+  const modSocket = modSocketFor(item);
+  const isModdedStat = modSocket
+    ? _.keys(modSocket.plug.stats).includes(String(stat.statHash))
```

Comment:
Use `Object.keys` instead of `_.keys`. But actually it's easier to just say `modSocket.plug.stats[stat.statHash] !== undefined`.